IfcPlusPlus/CMakeLists.txt

```diff
@@ -75,9 +74,14 @@ if (MSVC)
 	set_source_files_properties(src/ifcpp/IFC4X3/TypeFactory.cpp PROPERTIES COMPILE_FLAGS /bigobj)
 endif()
 
-add_library(IfcPlusPlus STATIC ${IFCPP_SOURCE_FILES})
+add_library(IfcPlusPlus ${IFCPP_SOURCE_FILES})
+
+if(NOT BUILD_SHARED_LIBS)
+    target_compile_definitions(IfcPlusPlus PUBLIC IFCQUERY_STATIC_LIB)
+endif()
 
 TARGET_INCLUDE_DIRECTORIES(IfcPlusPlus
+    PUBLIC "$<INSTALL_INTERFACE:include;include/ifcpp/IFC4X3/include;include/ifcpp/external>"

```

Comment:
This is required for proper `INTERFACE_INCLUDE_DIRECTORIES` in exported cmake config.